cypress/views/pvc.ts

```diff
@@ -15,7 +15,10 @@ export const pvc = {
       cy.byTestID('Block-radio-input').click();
     }
     cy.byTestID('create-pvc').click();
-    cy.contains('Bound', { timeout: 10000 });
+    cy.contains('Bound', { timeout: 30000 });
+  },
+  deleteIfExists: (pvcName: string, namespace: string) => {
+    cy.exec(`oc delete pvcs ${pvcName} -n ${namespace}`);
```

Comment:
won't it give error if PVC not found ?
```suggestion
    cy.exec(`oc delete pvcs ${pvcName} -n ${namespace}`, {failOnNonZeroExit: false });
```

Reply:
I have made the changes. thanks